tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
+#          through libyui-rest-client.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use YuiRestClient::Logger;
+
+sub run {
+    # in Disk Activation page press "ZFCP" button
+    my $disk_activation_selector = $testapi::distri->get_disk_activation_selector();
+    $disk_activation_selector->zfcp();
+
+    # in the ZFCP Device overview press the "Add" button
+    my $zfcp_configuration_overview = $testapi::distri->get_zfcp_configuration_overview();
+    $zfcp_configuration_overview->add();
+
+    # In the Add Device page insert hard wired channel id and press next
+    my $zfcp_add_disk = $testapi::distri->get_zfcp_add_disk();
+    $zfcp_add_disk->set_channel("0.0.fa00");
+    $zfcp_add_disk->next();
+
+    # yui.log from 1st VR showed that it takes a while for
+    # device detection so lets sleep a bit.
+    sleep(8);
+
+    # Now we should be back in ZFCP Device overview page
+    # we read the whole table with devices from libyui
+    my @devtable = $zfcp_configuration_overview->get_devices();
+
+    # Now lets extract the drive letters (a..d) from the table
+    my @devs = ();
+    for my $row (@devtable) {
+        push(@devs, substr @$row[3], -1);
+    }
+    @devs = sort(@devs);
+
+    # We expect drive letters from 'a..d'
+    my @expectdevs = ("a" .. "d");
+
+    # Now do the tests
+    if (@devs != @expectdevs) {
+        YuiRestClient::Logger->get_instance()->error("Number of found devices does not match expectation.");
+        die "Device number mismatch";
+    }
+
+    my $mismatchcount = 0;
+    for (my $i = 0; $i < @devs; $i++) {
+        if ($devs[$i] ne $expectdevs[$i]) {
+            $mismatchcount++;
+          YuiRestClient: Logger->get_instance()->error("$devs[$i] does not match $expectdevs[$i]");
+        }
+    }
+    if ($mismatchcount > 0) {
+        die "Wrong device names returned";
+    }
+
+    # If we reach it here then everything is fine
+    # Leave ZFCP Configuration Overview.
+    $zfcp_configuration_overview->next();
```

Comment:
here we should go with two tests as well, `accept_configured_zfcp_devices` and `finish_disk_activation`.